src/video/playVideoFrames.m

```diff
@@ -23,7 +33,7 @@ function playVideoFrames(video, cfg)
     % frames presentation loop
     for f = 1:video.frame.number
 
-        Screen('DrawTexture', cfg.screen.win, video.frames(f).videoFrameTexture, [], [], 0);
+        Screen('DrawTexture', cfg.screen.win, video.frames(f).videoFrameTexture, [], [video.position], 0);
```

Comment:
Quick suggestion:  do you think you could give a name to the empty and 0 value in here.

```
Screen('DrawTexture', cfg.screen.win, video.frames(f).videoFrameTexture, [], [video.position], 0);
```

Otherwise if I want to figure out what those arguments are I have to go and read the help of PTB (which is a pain to read because of the horrible formatting). We are not doing this consistently in CPP_PTB but we should try to move in this direction.

Maybe something like this?

```
sourceRect = [];
rotationAngle = 0;
Screen('DrawTexture', cfg.screen.win, video.frames(f).videoFrameTexture, sourceRect, video.position, rotationAngle);
```